ci-operator/step-registry/ipi/conf/inframachineset/ipi-conf-inframachineset-commands.sh

```diff
@@ -161,9 +161,6 @@ data:
     prometheusOperator:
       nodeSelector:
         node-role.kubernetes.io/infra: ""
-    grafana:
```

Comment:
Why is this being removed?

Reply:
"Grafana component in monitoring stack" has removed from 4.11, see [doc](https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-deprecated-removed-features), the step "ipi-conf-inframachineset" will hit error "unknown field "grafana" from 4.18, so removing it